### ISSUE TITLE
bin/grav install fails with message

### ISSUE THREAD

**Issue by fantasticme** [getgrav/grav#540]
```php
Fatal error: Class Grav\Common\Errors\Errors may not inherit from final class (Whoops\Run) in grav/system/src/Grav/Common/Errors/Errors.php on line 14
```

**Comment by flaviocopes** [comment]
Same as https://github.com/getgrav/grav/issues/536 ? Are you running the official release?

**Comment by fantasticme** [comment]
I am running the latest build with
```php
git clone https://github.com/getgrav/grav.git
```

**Comment by flaviocopes** [comment]
Read what I wrote on the other commit. There's a PR in place to fix this error, but it's not yet merged because we need to drop PHP 5.4 because of changes in the Whoops library, and we need to prepare for that.

You can use that branch now (https://github.com/getgrav/grav/pull/527), or just checkout the latest 1.0.5 release instead of develop.

Unless for development, I'd advise not to clone from Git directly, since Grav has an update mechanism already built-in.
